### ISSUE TITLE
Increased Abdominal circumference 

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5891]
**Preferred term label:** Increased Abdominal circumference 

**Synonyms** Enlarged Abdomen; Increased AC

**Definition (free text, please give PubMed ID)** Circumference of the abdomen is larger than the normal range at a given age., PMID: 26611149

**Parent term (use hpo.jax.org/app)** Abnormality of the abdominal wall HP:0004298

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
According to the cited article, " Neonatal abdominal circumference is not routinely measured, and no reference values for this measurement have been determined." 
Therefore, I do not think this is a good HPO term, since there is no clear way of stating whether the abdomen circumference is abnormal. There are many related HPO terms to describe related features.
